classes/Cart.php

```diff
@@ -3233,27 +3233,29 @@ public function getTotalShippingCost($delivery_option = null, $use_tax = true, C
         if (isset(Context::getContext()->cookie->id_country)) {
             $default_country = new Country(Context::getContext()->cookie->id_country);
         }
-        if (null === $delivery_option) {
+        if (is_null($delivery_option)) {
```

Comment:
We worked on the whole project to have a comparison instead of `is_null`. Please bring back the old version.

```suggestion
        if (inull === $delivery_option) {
```